src/java.base/share/classes/java/util/LinkedList.java

```diff
@@ -1457,10 +1507,18 @@ public int lastIndexOf(Object o) {
             return rlist.lastIndexOf(o);
         }
 
+        public int findLastIndex(Predicate<? super E> filter) {
+            return rlist.lastIndexOf(filter);
+        }
+
         public int indexOf(Object o) {
             return rlist.indexOf(o);
         }
 
+        public int findIndex(Predicate<? super E> filter) {
+            return rlist.indexOf(filter);
```

Comment:
Should be `findIndex`? Similarly with `findLastIndex`.

Reply:
> Should be `findIndex`? Similarly with `findLastIndex`.

Yeah i guess IDE didnt do a great job with refactor. I will review aall changes manually a bit later